ci/azure-piplines.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       source ./ci/_activate_current_env.sh
       pip install --no-deps .
       cd ../hera_cal
-      pip install --no-deps .
```

Comment:
We recently changed `hera_cal` to simply install in a virtual environment without conda for CI, using the direct dependencies listed in `setup.cfg`. Thus, the conda yaml in the hera_cal repo is not up to date, and actually should probably be removed because it's not used. The reason for doing this was speed.

Reply:
ok, then I think the entire action should be updated not to use that conda file.

Reply:
Ah, I forgot about this comment, sorry! Yeah, let's make this an issue, because I think it's unrelated to this PR.

Reply:
See https://github.com/RadioAstronomySoftwareGroup/pyuvdata/issues/1293